scikeras/wrappers.py

```diff
@@ -697,34 +701,46 @@ def feature_encoder(self):
         """
         return FunctionTransformer()
 
-    def fit(self, X, y, sample_weight=None) -> "BaseWrapper":
+    def fit(self, X, y, sample_weight=None, **kwargs) -> "BaseWrapper":
         """Constructs a new model with ``model`` & fit the model to ``(X, y)``.
 
         Parameters
         ----------
-        X : Union[array-like, sparse matrix, dataframe]of shape (n_samples, n_features)
+        X : Union[array-like, sparse matrix, dataframe] of shape (n_samples, n_features)
             Training samples, where n_samples is the number of samples
             and n_features is the number of features.
         y : Union[array-like, sparse matrix, dataframe] of shape (n_samples,) or (n_samples, n_outputs)
             True labels for X.
         sample_weight : array-like of shape (n_samples,), default=None
             Array of weights that are assigned to individual samples.
             If not provided, then each sample is given unit weight.
-
+        **kwargs : Dict[str, Any]
+            Extra arguments to route to ``Model.fit``.
+            This functionality has been deprecated, and will be removed in SciKeras 1.0.0.
+            These parameters can also be specified by prefixing `fit__` to a parameter at initialization;
+            e.g, `BaseWrapper(..., fit__batch_size=32, predict__batch_size=1000)`.
         Returns
         -------
         BaseWrapper
             A reference to the instance that can be chain called
             (ex: instance.fit(X,y).transform(X) )
         """
+        for k, v in kwargs.items():
+            warnings.warn(
+                "``**kwargs`` has been deprecated in SciKeras 0.2.1, and will be 1.0.0."
```

Comment:
```suggestion
                "``**kwargs`` has been deprecated in SciKeras 0.2.1 and support will be removed be 1.0.0."
```

This goes for several other places too.

Reply:
Thanks, I replaced it in 7dd1cd4.